site/src/modules/workspaces/WorkspaceTiming/WorkspaceTimings.tsx

```diff
@@ -135,19 +158,18 @@ export const WorkspaceTimings: FC<WorkspaceTimingsProps> = ({
 							)}
 
 						{view.name === "detailed" &&
-							view.category.id === "workspaceBoot" && (
+							view.stage.section !== "provisioning" && (
```

Comment:
this could be put in a ternary with the condition above to reduce repeated conditions